server/src/seeders/20210111120513-demo-data-set.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const v4 = require('uuid').v4;
+const bcrypt =  require('bcryptjs');
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        const date = new Date();
+        const org_id_svm = v4();
+        const user_id_svm_admin = v4();
+        const team_id_svm_outdoor = v4();
+        const role_id_svm_admin = v4();
+        const organizations = [{
+            id: org_id_svm,
+            access_code: 'svm2020',
+            config: '{}',
+            title: 'SVM',
+            is_active: true,
+            admin_role_id: role_id_svm_admin,
+            createdAt: date,
+            updatedAt: date
+        }];
+        queryInterface.bulkInsert('Organizations', organizations, {});
+        
+        const roles = [{
+            id: role_id_svm_admin,
+            title: 'SVM Admin',
+            user_deletable: false,
+            is_active: true,
+            org_id: org_id_svm,
+            createdAt: date,
+            updatedAt: date
+        }];
+        queryInterface.bulkInsert('Roles', roles, {});
+
+        const teams = [{
+            id: team_id_svm_outdoor,
+            title: 'SVM Outdoor',
+            is_active: true,
+            org_id: org_id_svm,
+            maintain_role_id: role_id_svm_admin,
+            createdAt: date,
+            updatedAt: date
+        }];
+        queryInterface.bulkInsert('Teams', teams, {});
+
+        const roleAssignments = [{
+            id: v4(),
+            role_id: role_id_svm_admin,
+            user_id: user_id_svm_admin,
+            createdAt: date,
+            updatedAt: date
+        }];
+        queryInterface.bulkInsert('RoleAssignments', roleAssignments, {});
+        
+        const memberships = [{
+            id: v4(),
+            user_id: user_id_svm_admin,
+            team_id: team_id_svm_outdoor,
+            createdAt: date,
+            updatedAt: date
+        }];
+        queryInterface.bulkInsert('Memberships', memberships, {});
+        
+        const SALT_FACTOR = 10;
+        const hashedPassword = await bcrypt.hash('admin123', SALT_FACTOR);
+        const users = [{
+            id: user_id_svm_admin,
+            email: 'admin@admin.com',
+            username: 'svm_admin',
```

Comment:
Wenn schon personalisiert :)
```suggestion
            username: 'kehren',
```